stonesoup/updater/ensemble.py

```diff
@@ -118,19 +114,18 @@ def predict_measurement(self, predicted_state, measurement_model=None,
         measurement_model : :class:`~.MeasurementModel`
             The measurement model. If omitted, the model in the updater object
             is used
-
```

Comment:
Some of the blank lines are required, as they are breaking formatting in some places.